packages/docs-reanimated/docs/core/makeMutable.mdx

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_position: 8
+---
+
+# makeMutable
+
+:::caution
+The usage of `makeMutable` is discouraged in most cases. It's recommended to use the [`useSharedValue`](/docs/core/useSharedValue) hook instead unless you know what you're doing and you are aware of the consequences (see the [Remarks](#remarks) section).
+
+`makeMutable` is used internally and its behavior may change over time.
+:::
+
+`makeMutable` is a function internally used by the [`useSharedValue`](/docs/core/useSharedValue) hook to create a [shared value](/docs/fundamentals/glossary#shared-value).
+
+It makes it possible to create mutable values without the use of the hook, which can be useful in some cases (e.g. in the global scope, as an array of mutable values, etc.).
+
+The created object is, in fact, the same as the one returned by `useSharedValue` hook, so the further usage is the same.
+
+:::info
+We use `mutable value` name for an object created by `makeMutable` to distinguish it from the `shared value` created by `useSharedValue`. Technically, `shared value` is a `mutable value` with an automatic cleanup.
+:::
+
+## Reference
+
+```javascript
+import { makeMutable } from 'react-native-reanimated';
+
+const mv = makeMutable(100);
+```
+
+<details>
+<summary>Type definitions</summary>
+
+```typescript
+interface SharedValue<Value = unknown> {
+  value: Value;
+  addListener: (listenerID: number, listener: (value: Value) => void) => void;
+  removeListener: (listenerID: number) => void;
+  modify: (
+    modifier?: <T extends Value>(value: T) => T,
+    forceUpdate?: boolean
+  ) => void;
+}
+
+function makeMutable<Value>(initial: Value): SharedValue<Value>;
+```
+
+</details>
+
+### Arguments
+
+#### `initial`
+
+The value you want to be initially stored to a `.value` property. It can be any JavaScript value like `number`, `string` or `boolean` but also data structures such as `array` and `object`.
+
+### Returns
+
+`makeMutable` returns a mutable value with a single `value` property initially set to the `initial`.
+
+Values stored in mutable values can be accessed and modified by their `.value` property.
+
+## Example
+
+import MakeMutable from '@site/src/examples/MakeMutable';
+import MakeMutableSrc from '!!raw-loader!@site/src/examples/MakeMutable';
+
+<InteractiveExample src={MakeMutableSrc} component={MakeMutable} showCode />
+
+## Remarks
+
+- All remarks from the [useSharedValue](/docs/core/useSharedValue) hook apply to `makeMutable` as well.
+
+- Don't use `makeMutable` directly in the component scope. When component re-renders, it will create the completely new object (with the new `initial` value if it was changed) and the state of the previous mutable value will be lost.
```

Comment:
```suggestion
- Don't call `makeMutable` directly in the component scope. When component re-renders, it will create the completely new object (with the new `initial` value if it was changed) and the state of the previous mutable value will be lost.
```